modules/libcom/src/error/errSymLib.c

```diff
@@ -109,21 +92,48 @@ static epicsUInt16 errhash(long errNum)
 
     modnum = (unsigned short) (errNum >> 16);
     errnum = (unsigned short) (errNum & 0xffff);
-    return (((modnum - 500) * 20) + errnum) % NHASH;
+    return ((modnum - (MIN_MODULE_NUM - 1)) * 20 + errnum) % NHASH;
 }
 
 /****************************************************************
  * ERRSYMBOLADD
- * adds symbols to the master errnumlist as compiled from errSymTbl.c
+ * adds symbols to the global error symbol hash table
  ***************************************************************/
 int errSymbolAdd(long errNum, const char *name)
 {
-    ERRNUMNODE *pNew = (ERRNUMNODE*) callocMustSucceed(1,
-        sizeof(ERRNUMNODE), "errSymbolAdd");
+    ERRNUMNODE *pNextNode = NULL;
+    ERRNUMNODE **phashnode = NULL;
+    ERRNUMNODE *pNew = NULL;
+    int modnum = (epicsUInt16) (errNum >> 16);
+
+    if (modnum < MIN_MODULE_NUM)
+        return S_err_invCode;
+
+    initErrorHashTable();
+
+    epicsUInt16 hashInd = errhash(errNum);
```

Comment:
The appveyor vs2012 build is still failing on this line.  My attempt to include `epicsTypes.h` above did not help, which confuses me.  I am not certain why this does not include the typedef for `epicsUInt16`?

```
cl               -nologo -FC -D__STDC__=0 -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE -DBUILDING_libCom_API   -Ox -GL -Oy-   -W3        -MD -DEPICS_BUILD_DLL -DEPICS_CALL_DLL    -I. -I../O.Common -I. -I../osi/compiler/msvc -I../osi/compiler/default -I. -I../osi/os/WIN32 -I../osi/os/default -I.. -I../as -I../bucketLib -I../calc -I../cvtFast -I../cppStd -I../cxxTemplates -I../dbmf -I../ellLib -I../env -I../error -I../fdmgr -I../flex -I../freeList -I../gpHash -I../iocsh -I../log -I../macLib -I../misc -I../osi -I../pool -I../ring -I../taskwd -I../timer -I../yacc -I../yacc -I../yajl -I../../../../include/compiler/msvc -I../../../../include/os/WIN32 -I../../../../include         -c ../error/errSymLib.c
errSymLib.c
c:\projects\epics-base\modules\libcom\src\error\errsymlib.c(114) : error C2275: 'epicsUInt16' : illegal use of this type as an expression
        c:\projects\epics-base\modules\libcom\src\misc\epicstypes.h(47) : see declaration of 'epicsUInt16'
c:\projects\epics-base\modules\libcom\src\error\errsymlib.c(114) : error C2146: syntax error : missing ';' before identifier 'hashInd'
c:\projects\epics-base\modules\libcom\src\error\errsymlib.c(114) : error C2065: 'hashInd' : undeclared identifier
c:\projects\epics-base\modules\libcom\src\error\errsymlib.c(117) : error C2065: 'hashInd' : undeclared identifier
make[4]: *** [../../../../configure/RULES_BUILD:258: errSymLib.obj] Error 2
make[4]: Leaving directory 'C:/projects/epics-base/modules/libcom/src/O.win32-x86'
make[4]: *** Waiting for unfinished jobs....
make[4]: Entering directory 'C:/projects/epics-base/modules/libcom/src/O.win32-x86'
cl               -nologo -FC -D__STDC__=0 -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE -DBUILDING_libCom_API   -Ox -GL -Oy-   -W3        -MD -DEPICS_BUILD_DLL -DEPICS_CALL_DLL    -I. -I../O.Common -I. -I../osi/compiler/msvc -I../osi/compiler/default -I. -I../osi/os/WIN32 -I../osi/os/default -I.. -I../as -I../bucketLib -I../calc -I../cvtFast -I../cppStd -I../cxxTemplates -I../dbmf -I../ellLib -I../env -I../error -I../fdmgr -I../flex -I../freeList -I../gpHash -I../iocsh -I../log -I../macLib -I../misc -I../osi -I../pool -I../ring -I../taskwd -I../timer -I../yacc -I../yacc -I../yajl -I../../../../include/compiler/msvc -I../../../../include/os/WIN32 -I../../../../include         -c errSymTbl.c
errSymTbl.c
```

https://ci.appveyor.com/project/epics-base-7/epics-base/builds/47895623/job/6a76h0mefu03jsr7#L1429